### PR TITLE
Convert has_lib.sh to sh so it also works on BSD OSes

### PR DESCRIPTION
**Pull request by hakkiplaten**
Passes http://www.shellcheck.net/

Ref. #754

### CODE REVIEW

File: util/has_lib.sh [hunk 1]
```diff
@@ -52,7 +53,7 @@ case "$1" in
     result=$?
     ;;
   *)
-    >&2 echo "Unknown library: $1"
+    echo "Unknown library: $1"
```
Comment (LinusU) [inline util/has_lib.sh]: Is there no way to print to stderr in `sh`?

Comment (hakkiplaten) [inline util/has_lib.sh]: Was a bug at ShellCheck. That line is valid. I've fixed the PR.

**Comment by hakkiplaten** [comment]
@LinusU all good to merge?

**Comment by LinusU** [comment]
Thank you for this

**Comment by LinusU** [comment]
Backporting this to 1.x in #826

**Comment by LinusU** [comment]
Published as 1.6.1

**Pull request merged**